src/ai/AiModelManager.ts

```
@@ -1,52 +1,76 @@
+import { SparkAiModel } from "./spark/SparkAiModel.ts";
+import { WenXinAiModel } from "./wenxin/WenXinAiModel.ts";
 
-export class AiModelManager {
+export namespace AiModelManager {
+    const models = new Map<string, AiModel>();
 
-    private static models: Record<string, AiModel> = {};
-
-    static init(editor: InnerEditor, globalConfig: AiGlobalConfig) {
+    export function registerModels(editor: InnerEditor, globalConfig: AiGlobalConfig) {
         if (globalConfig && globalConfig.models) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)